src/main/java/uk/gov/justice/digital/service/datareconciliation/model/CurrentStateCountTableResult.java

```diff
@@ -0,0 +1,46 @@
+package uk.gov.justice.digital.service.datareconciliation.model;
+
+import lombok.Getter;
+
+/**
+ * Represents the results of running the total counts data reconciliation for a single "current state" table in DataHub
+ * replicated across multiple datastores
+ */
+@Getter
+public class CurrentStateCountTableResult {
+    private final long nomisCount;
+    private final long structuredCount;
+    private final long curatedCount;
+    // A null value indicates that there is no Operational DataStore count
```

Comment:
I really wanted to use Optional to be more explicit but Java has a convention that Optional should not be used to represent absence vs presence in fields.